chainer/cuda.py

```diff
@@ -82,11 +82,13 @@ def check_cuda_available():
                '(see https://github.com/pfnet/chainer#installation).')
         msg += str(_resolution_error)
         raise RuntimeError(msg)
-    if not cudnn_enabled:
+    if (not cudnn_enabled and
+            not getattr(check_cuda_available, '_already_warned', False)):
```

Comment:
To eliminate `getattr`, `check_cuda_available._already_warned = False` should be executed before any call of `check_cuda_available`. Is it simple?

Alternatively, I can use a global variable instead, but it seems unclear for a reader. `_already_warned` should be belong to `check_cuda_available` for readability.